inc/saibuffer.h

```diff
@@ -150,6 +150,12 @@ typedef enum _sai_ingress_priority_group_stat_t
     /** Get dropped packets count [uint64_t] */
     SAI_INGRESS_PRIORITY_GROUP_STAT_DROPPED_PACKETS = 0x00000008,
 
+    /** Get current pg occupancy in cells [uint64_t] */
+    SAI_INGRESS_PRIORITY_GROUP_STAT_CURR_OCCUPANCY_CELLS = 0x00000009,
+
+    /** Get watermark pg occupancy in cells [uint64_t] */
+    SAI_INGRESS_PRIORITY_GROUP_STAT_WATERMARK_CELLS = 0x0000000a,
```

Comment:
Hi Marian, just a sanity checks here, do we need:
- SAI_INGRESS_PRIORITY_GROUP_STAT_XOFF_ROOM_CURR_OCCUPANCY_CELLS
- SAI_INGRESS_PRIORITY_GROUP_STAT_XOFF_ROOM_WATERMARK_CELLS

For SAI_INGRESS_PRIORITY_GROUP_STAT_BYTES, I believe it is supported, and we don't need the cell counts (as it will be bad for tracking the traffic volume).

Reply:
are we missing the buffer pool ones? or can they be supported in bytes?